README.md

```diff
@@ -28,11 +28,11 @@ Here are a few things that were removed but did not have deprecation warnings in
 
 ## Automatic Node.js Polyfills Removed
 
-In the early days, webpack's aim was to allow running most node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
+In the early days, webpack's aimed to allow running most node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
 
 While this makes using modules written for node.js easy, it adds these huge polyfills to the bundle. In many cases these polyfills are unnecessary.
 
-webpack 5 stops automatically polyfilling these core modules and focuses on frontend-compatible modules.
+webpack 5 stops automatically polyfilling these core modules and focus on frontend-compatible modules.
```

Comment:
I'm not sure how the current version is a typo. It looks grammatically correct to me!

Reply:
After a little bit of searching, I found out both versions are correct.
I will change it back to `focuses`